test/experiments/assert-labels-and-phase-are-set.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: dope.mayadata.io/v1
+kind: Recipe
+metadata:
+  name: assert-absence-of-deprecated-daemonsets
+  namespace: d-testing
+  labels:
+    d-testing.dope.mayadata.io/inference: "true"
```

Comment:
remove this label, since inference targets the primary recipes & not the helper recipes.

Reply:
add some specific labels e.g. 
```yaml
labels:
  assert-phase-is-set-in-label: true
```